google/cloud/__init__.py

```diff
@@ -14,11 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import List
+
 try:
     import pkg_resources
 
     pkg_resources.declare_namespace(__name__)
 except ImportError:
     import pkgutil
 
-    __path__ = pkgutil.extend_path(__path__, __name__)
+    __path__: List[str] = pkgutil.extend_path(
+        __path__, __name__
+    )  # pytype: disable=annotation-type-mismatch
```

Comment:
Given that `mytype` and `pytype` disagree, I think I'd prefer just `# type: ignore` here.  *Nobody* is going to care in the wild, anyway.